tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -68,15 +68,15 @@
  * The expected parity is even so that 0 is considered a valid encoding.
  *
  * Return 0 if value has even parity and a nonzero value otherwise. */
-int check_parity( uint32_t value )
+int has_even_parity( uint32_t value )
```

Comment:
I like the new name, but it seems in opposition to the comment above. Which made me notice: either I'm very confused right now, or for the past few round of reviews we all missed that the comment on the line before is wrong. Clearly this function will return `1` when passed `0`, which has even parity, so I think the comment needs updating.

Reply:
Just how hard can it be to return a boolean value from a function...